crates/sui-gateway/src/api.rs

```diff
@@ -101,7 +39,11 @@ pub trait RpcGateway {
     /// Synchronize client state with validators.
     #[method(name = "syncAccountState")]
     async fn sync_account_state(&self, address: SuiAddress) -> RpcResult<()>;
+}
 
+#[open_rpc(namespace = "sui", tag = "Read API")]
+#[rpc(server, client, namespace = "sui")]
+pub trait RpcReadApi {
```

Comment:
Now that full nodes and validators are being merged, and the full node will become THE RPC server of the future, I would imagine we could merge `RpcReadApi` and `RpcFullNodeReadApi` correct?

Reply:
We can merge them as soon as we retire RPC Gateway!